application/blueprints/auth/views.py

```
@@ -32,11 +76,13 @@ def authorize():
         # check if user is a member of digitial-land org - if they are the members endpoint
         # will return status code 204
         # https://docs.github.com/en/rest/orgs/members?apiVersion=2022-11-28#check-organization-membership-for-a-user
-        url = (
-            f"https://api.github.com/orgs/digital-land/members/{user_profile['login']}"
-        )
+        github_api_base_url = current_app.config["GITHUB_API_BASE_URL"]
+        url = f"{github_api_base_url}/orgs/digital-land/members/{user_profile['login']}"
         resp = requests.get(url, headers=headers)
```

Ruff (0.15.18)

[error] Probable use of `requests` call without timeout

(S113)